guides/keras_nlp/getting_started.py

```diff
@@ -128,6 +128,13 @@
 # Format is (review text tensor, label tensor)
 print(imdb_train.unbatch().take(1).get_single_element())
 
+# The above line does the
+# following trick.
+for data_elem in imdb_train:
+    print(data_elem[0][0])
+    print(data_elem[1][0])
```

Comment:
Formatting a string that adds more details about the data being printed would help readability

Reply:
Done the required changes.